server/microbench_test.go

```diff
@@ -0,0 +1,285 @@
+package server
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"io"
+	"runtime"
+	"sync"
+	"testing"
+
+	"github.com/cloudflare/gokeyless/internal/test/params"
+	"github.com/joshlf/testutil"
```

Comment:
@lziest has concerns about vendoring too much, and this is only in a test, so I figured it's fine to just let people use 'go get' to run the tests. There are a number of other test dependencies that are also not vendored. I'd also be happy to vendor it, though.